scheduler/resource/task.go

```diff
@@ -173,6 +182,10 @@ func NewTask(id, url string, taskType commonv1.TaskType, meta *commonv1.UrlMeta,
 				t.UpdateAt.Store(time.Now())
 				t.Log.Infof("task state is %s", e.FSM.Current())
 			},
+			TaskEventLeave: func(e *fsm.Event) {
+				t.UpdateAt.Store(time.Now())
+				t.Log.Infof("task state is %s", e.FSM.Current())
```

Comment:
Why not print more info, like:
```
t.Log.Infof("event is TaskEventLeave,  task state is %s", e.FSM.Current())
```

Reply:
The log remains consistent, and I don't care what event happens when I check, only the state switch.